keyboards/cool836A/config.h

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2021 Ohashi
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0xFEED
+#define PRODUCT_ID   0x0000
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Ohashi
+#define PRODUCT      cool836A
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 6
```

Comment:
These lines makes it difficult for me to describe cool836A as a 3x12 keyboard.

Reply:
These don't matter. They describe the electrical layout, whereas a 3x12 board describes the key layout.  Which aren't always the same.   

Plenty of boards do this, too.  Like the planck rev6 uses a square matrix, but is still considered 4x12.

Reply:
I'm surprised with your super quick replying. Thank you very much.

Reply:
Then, could you check out [another comment by myself](https://github.com/qmk/qmk_firmware/pull/11467#discussion_r553705632)

Reply:
I've read [this Reddit page](https://www.reddit.com/r/olkb/comments/8x7fk0/qmk_compile_error_when_adding_additional_columns/) and doubted that I have to change those definiton of `MATRIX_ROWS` and `MATRIX_COLS`

Reply:
> These don't matter. They describe the electrical layout, whereas a 3x12 board describes the key layout. Which aren't always the same.
> 
> Plenty of boards do this, too. Like the planck rev6 uses a square matrix, but is still considered 4x12.


Chances are that I should consult planck's codes.
I'll appreciate it if you wait for a little more time.